builtin/mainmenu/common.lua

```diff
@@ -234,7 +234,13 @@ end
 --------------------------------------------------------------------------------
 function asyncOnlineFavourites()
 
-	menudata.favorites = {}
+	if not menudata.public_known then
+		menudata.public_known = {{
+			name = "loading…",
+			description = "Try reenabling public serverlist and check if you have internet"
```

Comment:
Don't use an ASCII special character (i.e. : `…`) since it's not always supported by the OS (especially MacOS).

Also *Loading*, and not *loading*. You should translate those strings using S(" ").